client/cypress/support/commands.js

```diff
@@ -13,8 +13,32 @@
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
 Cypress.Commands.add('addListing', (meals, desc) => {
-  cy.visit('http://localhost:3000/listings/new');
+  cy.get('#new-listing-link').click();
   cy.get('#meals').type(meals);
   cy.get('#desc').type(desc);
-  cy.get('.btn').click();
+  cy.get('#create-listing-btn').click();
+});
+
+Cypress.Commands.add('charityLogin', () => {
+  cy.visit('/login');
+  cy.get('#username').type('Charity Supreme');
+  cy.get('#password').type('Charity123!');
+  cy.get('#loginSubmit').click();
+});
+
+Cypress.Commands.add('restaurantLogin', () => {
+  cy.visit('/login');
+  cy.get('#username').type('Restaurant Extreme');
+  cy.get('#password').type('Password123!');
+  cy.get('#loginSubmit').click();
+});
+
+Cypress.Commands.add('switchToCharity', () => {
```

Comment:
I'm wondering though that these 'switch to' methods aren't super reusable, they rely on certain tests/cy commands coming before? I believe good testing we're supposed to 'tear down' any set up at the end of tests so maybe we should be logging out at the end of each and only using log ins rather than switch to's